### PR TITLE
Add a PowerDNS pipeline

### PR DESCRIPTION
**Pull request by ekohl**
This is semi work in progress. It includes https://github.com/theforeman/forklift/pull/405 and is currently limited to MySQL backend support. Ideally it would also support postgresql and rest.

### CODE REVIEW

File: pipelines/pipeline_smartproxy_powerdns.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  become: yes
+  vars:
+    bats_forklift_dir: /vagrant
+    bats_update_forklift: "no"
+    bats_tests:
+      - "fb-proxy-dns.bats"
+    foreman_installer_options: "
+       --no-enable-foreman
+       --no-enable-foreman-cli
+       --no-enable-foreman-plugin-setup
```
Comment (ehelms) [inline pipelines/pipeline_smartproxy_powerdns.yml]: Heh -- guess there still isn't a proper 'foreman proxy' scenario?

Comment (ekohl) [inline pipelines/pipeline_smartproxy_powerdns.yml]: I didn't consider looking at that, but it turns out there isn't one :)

**Comment by ehelms** [comment]
@ekohl Any reason not to merge this?

**Comment by ekohl** [comment]
@ehelms it got lost in the many github notifications.

**Pull request merged**